README.md

```diff
@@ -73,3 +73,26 @@ This cookbook includes automatic detection and tagging of your meter with variou
 #### OpsWorks
 
 If you are using OpsWorks this cookbook should work out of the box (with the above dependencies). This cookbook also includes automatic detection and tagging of your meter with layers, stack name and applications if any exist.
+
```

Comment:
This functionality should be transparent to the user. Checking to see if a databag exists or not is trivial:
data_bag('boundary').include?("meter") == true do
  merge

There is no need to allow the user to define whether to merge or the name or item as we own the namespace.
Name should be: boundary
Item should be: meter